website/content/docs/release-notes/nomad/v1_8_x.mdx

```diff
@@ -0,0 +1,39 @@
+---
+layout: docs
+page_title: 1.8.x
+description: >-
+  Nomad release notes for version 1.8.x
+---
+
+# Nomad 1.8.0
+
+We are pleased to announce the following Nomad updates.
+
+## Release highlights
+
+- **Long Term Support Release (Enterprise):** Nomad 1.8.0 now introduces a long term supported release which provides extended maintenance (two years of critical fixes provided through minor releases and efficient upgrades (support for direct upgrades from one LTS release to the next). 
+
+- **Exec2 Task Driver (beta):** The `exec2` driver is used to execute a command for a task. It offers a security model optimized for running 'ordinary' processes with very low startup times and minimal overhead in terms of CPU, disk, and memory utilization. The `exec2` driver leverages kernel features such as the Landlock LSM, cgroups v2, and the unshare system utility. Due these enhancements, tasks no longer require leveraging filesystem isolation based on chroot, which provide an enhanced security profile and improved performance for the Nomad operator. Refer to [Exec2 Task Driver](/nomad/plugins/v1.8.x/drivers/exec2) for more details. 
+
+
+
+## What's deprecated
+
+- **LXC Task driver:** The [Community LXC Task Driver](/nomad/plugins/drivers/community/lxc) is now deprecated and will be removed in Nomad 1.9. Users are encouraged to migrate their workloads to another task driver such as the built-in Docker task driver[/nomad/docs/drivers/docker]. 
+- **ECS Task driver:** The remote [ECS Task Driver](https://developer.hashicorp.com/nomad/plugins/drivers/remote/ecs) is now deprecated and will be removed in Nomad 1.9. Users are encouraged to migrate their workloads to Amazon ECS or to another Nomad task driver. 
+
+## Upgrading
+
+For more detailed information, please refer to the [upgrade details page](/nomad/docs/upgrade/upgrade-specific) and the changelogs.
```

Comment:
I'm wondering if this should be "by reference" or whether we should simply combine the two?

Reply:
Sorry I don't quite follow you here. Could you elaborate happy to make it better. I was mostly following what Consul did here, not saying it is perfect.

Reply:
Oh just that we have the upgrade-specific page that includes a lot of what we have in these release notes page. Instead of spreading it out across two pages, maybe we could come up with a way to combine the two pages. (ex. put a "here's the hot new features" at the top, and then the list of upgrade notes).

Reply:
If it's ok, I'd like to just link to the version from the upgrade specific page for now.